org.pdtextensions.semanticanalysis/src/org/pdtextensions/semanticanalysis/validation/validator/VariableValidator.java

```diff
@@ -261,7 +263,7 @@ public int compare(VarComment o1, VarComment o2) {
 
 			@Override
 			public int compare(PHPDocBlock o1, PHPDocBlock o2) {
-				return o1.sourceStart() - o1.sourceStart();
```

Comment:
This was probably the initial intention (@Dawid you should confirm this please).